src/masonite/api/middleware/JWTAuthenticationMiddleware.py

```diff
@@ -0,0 +1,19 @@
+from masonite.middleware import Middleware
+from masonite.facades import Auth
+
+
+class JWTAuthenticationMiddleware(Middleware):
+    def before(self, request, response):
+        token = request.input("token")
+        if not token:
+            return response.view(
```

Comment:
Those views will be converted to json if json is requested I presume ? (I guess you fix that few days ago)

Reply:
yeah the view method does the response conversion